bazel/external/quiche.BUILD

```diff
@@ -108,7 +108,10 @@ envoy_cc_library(
     name = "http2_constants_lib",
     srcs = ["quiche/http2/http2_constants.cc"],
     hdrs = ["quiche/http2/http2_constants.h"],
-    copts = quiche_copt,
+    copts = select({
```

Comment:
Can you refactor them to `quiche_copt = select {...}` so you don't have to replace copt everywhere?

Reply:
Sadly, no. Select may not be nested.

Reply:
(We had attempted to do so earlier, a macro in bazel_build_system might be useful here, but the definition could not be abstracted during our attempt.)

Reply:
you're not nesting if you just make `select` to L55 and drop everywhere else?

Reply:
Thanks, our earlier attempts were apparently overly complex, I believe the current commit is the simplest representation you were asking us for?